spisea/atmospheres.py

```diff
@@ -913,6 +913,25 @@ def get_wd_atmosphere(metallicity=0, temperature=20000, gravity=4, verbose=False
         bbspec.convert('flam')
         bbspec *= (1000 * 3.08e18 / 6.957e10)**2
         return bbspec
+
+def get_bb_atmosphere(temperature=20000, verbose=False):
+    """
+    Return a blackbody spectrum
+
+    Parameters
+    ----------
+    temperature: float
+        The stellar temperature, in units of K
+    """
+    if verbose:
+        print('BB atmosphere')
+    
+    # Use a blackbody
+    bbspec = pysynphot.spectrum.BlackBody(temperature)
+    bbspec.convert('flam')
+    bbspec *= (1000 * 3.08e18 / 6.957e10)**2
```

Comment:
Can you add a comment about what these constants are and where they came from?

Reply:
Matt's comment... you will likely need the same inputs as the other atmosphere models (add metallicity, gravity, rebin etc... just ignore these.